api/v1alpha1/catalogentry_types.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	kcpv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/apis/v1alpha1"
+	conditionsv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/third_party/conditions/apis/conditions/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// These are valid conditions of CatalogEntry.
+const (
+	// CatalogEntryValid is a condition for CatalogEntry that reflects the validity
+	// of the referenced APIExport.
+	APIExportValidType conditionsv1alpha1.ConditionType = "APIExportValid"
+	// CatalogEntryInvalidReferenceReason is a reason for the CatalogEntryValid
+	// condition of APIBinding that the referenced CatalogEntry reference is invalid.
+	APIExportNotFoundReason = "APIExportNotFound"
+)
+
+//+kubebuilder:object:root=true
+//+kubebuilder:subresource:status
+//+kubebuilder:resource:scope=Cluster
+// CatalogEntry is the Schema for the catalogentries API
```

Comment:
```suggestion

// CatalogEntry is the Schema for the catalogentries API
```

Reply:
This makes sure the markers are not included in godoc output

Reply:
Done